tools/Makefile

```diff
@@ -281,6 +281,13 @@ READ_CMT= \
 # Reading cmt files
 $(call byte_and_opt,read_cmt,$(READ_CMT),)
 
+install::
+	if test -f read_cmt.opt; then \
```

Comment:
This should install a .byte version (as other tools are now) rather than just installing one - or for newly installed tools do we not care for both?